src/eg.js

```diff
@@ -67,18 +67,17 @@ eg.module("eg", ["jQuery", eg, window], function($, ns, global) {
 		// agent : null
 	};
 	/**
-	* Get browser agent information
-	*
-	* @ko Agent 정보를 반환한다. 값은 캐싱된다.
+	* Get the value of user-agent information the browser. Return value is cached.
+	* @ko 브라우저의 user-agent 정보를 반환한다. 반환 값을 캐시한다.
```

Comment:
@kishu I prefere @sculove opinion, because user mis-understand it as a his role.